### ISSUE TITLE
If a column value contains a semicolon (`;`), it cannot be processed correctly

### ISSUE THREAD

**Issue by noda-sin** [ing-bank/cassandra-jdbc-wrapper#50]
In [the code](https://github.com/ing-bank/cassandra-jdbc-wrapper/blob/a36a0d207be8fee1dbafc9aa731855bf649e9614/src/main/java/com/ing/data/cassandra/jdbc/CassandraStatement.java#L300), the query is split using a semicolon (`;`). Therefore, if a column value contains a semicolon, it could potentially break the query.
This issue was identified in the ticket below. 
https://github.com/liquibase/liquibase-cassandra/issues/258

Liquibase inserts strings separated by semicolons in the description into `DATABASECHANGELOG` table when there are multiple changes within a changeset.
However, it can't process correctly since query is split in the middle. 

error messages

```
Unexpected error running Liquibase: line 1:282 mismatched character '<EOF>' expecting ''' [Failed SQL: (0) INSERT INTO test.DATABASECHANGELOG (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, COMMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('create-tables', 'test', 'changelog.yml', 1703112425259, 1, '9:abd6cfa9eea355da8b0d16523bcc5115', 'sql; sql', '', 'EXECUTED', NULL, NULL, '4.25.0', '3112424168')]

Caused by: java.sql.SQLTransientException: com.datastax.oss.driver.api.core.servererrors.SyntaxError: line 1:282 mismatched character '<EOF>' expecting '''

cc @maximevw

**Comment by noda-sin** [comment]
maybe, this line intent to cure broken sql. but, not work properly.

https://github.com/ing-bank/cassandra-jdbc-wrapper/blob/a36a0d207be8fee1dbafc9aa731855bf649e9614/src/main/java/com/ing/data/cassandra/jdbc/CassandraStatement.java#L324-L328

**Comment by maximevw** [comment]
> maybe, this line intent to cure broken sql. but, not work properly.
> 
> https://github.com/ing-bank/cassandra-jdbc-wrapper/blob/a36a0d207be8fee1dbafc9aa731855bf649e9614/src/main/java/com/ing/data/cassandra/jdbc/CassandraStatement.java#L324-L328

Good point. The problem here is that code is not executed when the compliance mode "Liquibase" is active. So, I'll have to refactor the method `doExecute(String)` to split the statements correctly and build a list of valid CQL queries before executing them regardless of the compliance mode.

Thank you @noda-sin for pointing this out.

**Comment by maximevw** [comment]
Hello @noda-sin,

I released a version [4.11.1](https://github.com/ing-bank/cassandra-jdbc-wrapper/releases/tag/v4.11.1) to fix this issue. The artifact is also available in Maven Central.

A test with queries including semicolons in the inserted values has been added [here](https://github.com/ing-bank/cassandra-jdbc-wrapper/commit/3f059ab2afc0875b4f929130ae173562ddb36809#diff-56dd745721e5a9a2ddb430405ab4a7367044732bd7cb4424c235a0aa57b12ee0R256) and run both in default and Liquibase compliance modes. So, it seems now ok (the test indeed fails with the syntax error you mentioned when the fix is deactivated, but succeeds with the fix).

Please re-open this issue if the problem still persists.

**Comment by noda-sin** [comment]
@maximevw 
Wow! Thank you for handling despite the end of year.
Have a good New Year~ 🎉
